languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -22669,6 +22669,33 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- ====================================================================== -->
     <category id="COLLOQUIALISMS" name="Umgangssprache" type="register">
         <!-- TODO Make one Java-rule for context-independent, colloquial expressions and maintain a list -->
+            <rule id="KRASS_STIL" name="krass (erstaunlich, extrem)" default="temp_off" tags="picky">
+                <antipattern>
+                    <token regexp="yes">Unterschied|Gegensatz|Gegenteil|Widerspruch|Kontrast</token>
+                    <token skip="-1"/>
+                    <token>krass</token>
```

Comment:
Wenn du "krass" in beliebigem Abstand zum ersten Token brauchst, dann gehört das `skip="-1"` in das erste Token, und das zweite kann weg.